packages/plugin-annotation/src/index.ts

```diff
@@ -0,0 +1,11 @@
+import path from 'path';
+import type { RspressPlugin } from '@rspress/shared';
+
+const annotationComponent = path.resolve(__dirname, '../static/index.tsx');
```

Comment:
Should think of windows path, refer https://github.com/web-infra-dev/rspress/blob/main/packages/plugin-medium-zoom/src/index.ts